specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/expressRouteCrossConnection.json

```diff
@@ -799,12 +799,10 @@
     "ExpressRouteCrossConnectionProperties": {
       "properties": {
         "primaryAzurePort": {
-          "readOnly": true,
           "type": "string",
           "description": "The name of the primary port."
         },
         "secondaryAzurePort": {
-          "readOnly": true,
```

Comment:
@MyronFanQiu why aren't the readonly changes in this file seen as an SDK breaking change?  Has nothing been generated from this tag yet?

Reply:
Each month Network team would create a new branch containing new api-version. 2020-06-01 doesn't exist right now.